.github/workflows/deploy_doc.yml

```diff
@@ -53,7 +53,6 @@ jobs:
 
       - name: Deploy doc
         run: |
-          pip install -e .
```

Comment:
we had an issue in the past when we didn't prior install the package in the ci.

Reply:
it is installed above already